tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -0,0 +1,305 @@
+use crate::{
+    field::RecordFields,
+    fmt::{format, FormatEvent, FormatFields, MakeWriter},
+    layer::{Context, Layer},
+    registry::{LookupMetadata, LookupSpan, SpanRef},
+};
+use std::{cell::RefCell, fmt, io, marker::PhantomData};
+use tracing_core::{
+    span::{Attributes, Id, Record},
+    Event, Subscriber,
+};
+
+/// A [`Layer`] that logs formatted representations of `tracing` events.
+///
+/// [`Layer`]: ../trait.Layer.html
+#[derive(Debug)]
+pub struct FmtLayer<
```

Comment:
I think it would be ideal if this was called `fmt::Layer` rather than `fmt::Layer`, so that it's more consistent with `fmt::Subscriber`...but then we would need to use the layer trait as `layer::Layer`...